tests/test_boundaries.py

```diff
@@ -47,3 +47,28 @@ def test_equilibrium_boundary_pu(f_lattice):
     assert f.cpu().numpy() == pytest.approx(feq_field.cpu().numpy())
     #assert f.cpu().numpy() == pytest.approx(f_old.cpu().numpy())
 
+def test_anti_bounce_back_outlet(f_lattice):
```

Comment:
Please comment on what it does.

Reply:
Are the comments enough? Its pretty rough, but I was unsure what else to explain here.

Reply:
(I've added a reference to the book instead of "the source" now, please don't mind that part)